File: src/exemplos/5_EstruturasDeDados/4_Vetor/10-main.c
/**      @file: 10-main.c
 *     @author: Guilherme N. Ramos (gnramos@unb.br)
 * @disciplina: Algoritmos e Programação de Computadores
 *
 * Implementação de um programa que soma inteiros dados como
 * argumentos na linha de comando. */

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char** argv) {
  /* Assume que todos os argumentos são inteiros. */

  int i, soma = 0;
  for(i = 1; i < argc; ++i)
    soma += atoi(argv[i]);

  return soma;
}

File: src/exemplos/05_EstruturasDeDados/04_Vetor/10-main.py
#  -*- coding: utf-8 -*-
##   @package: 10-main.py
#     @author: Guilherme N. Ramos (gnramos@unb.br)
# @disciplina: Algoritmos e Programação de Computadores
#
# Implementação de um programa que soma inteiros dados como argumentos na linha
# de comando.


import sys


# Assume que todos os argumentos são inteiros.
soma = 0
for i in range(1, len(sys.argv)):
    soma += int(sys.argv[i])

sys.exit(soma)
